src/index.scss

```diff
@@ -26,3 +26,8 @@ main {
 .bx--content {
   padding: 0;
 }
+
+$feature-flags: (
+  ui-shell: true,
+  grid-columns-16: true,
+);
```

Comment:
The $feature-flags mapping should come before the styles import. See https://www.carbondesignsystem.com/tutorial/react-step-2/#install-grid

Reply:
Your Repos table width is not full width and your illustration on the landing page is not formatted properly. 
Please be sure to review your changes locally before pushing your updates.

Reply:
I'm sorry for the inconvenience. I fixed the style and verified the UI is formatted properly.